src/progress/web/method/session.i

```diff
@@ -0,0 +1,35 @@
+/*********************************************************************
+* Copyright (C) 2000 by Progress Software Corporation. All rights    *
+* reserved. Prior versions of this work may contain portions         *
+* contributed by participants of Possenet.                           *
+*  Per S Digre (pdigre@progress.com)                                 *
+*                                                                    *
+*********************************************************************/
+/*--------------------------------------------------------------------
+ File:    webutil/session.i
+ Purpose: Include for the session module
+ Updated: 04/04/98 pdigre@progress.com
+            Initial version
+          04/26/01 adams@progress.com
+            WebSpeed integration
+          05/23/01 mbaker@progress.com
+            Dynamics integration
+---------------------------------------------------------------------*/
+
+&IF DEFINED(session-included) = 0 &THEN
+&GLOBAL-DEFINE session-included YES
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) From src/progress/pct/pctWSComp.p - Preprocessor variable session-included should be SCOPED-DEFINED [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=rssw-oe-main%3Aeu.rssw.antlr.proparse.checks.GlobalDefine)